sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -123,6 +122,90 @@ export interface RuleProperties {
   action: SqlRuleAction;
 }
 
+/**
+ * Represents a Rule on a Subscription that is used to filter the incoming message from the
+ * Subscription.
+ */
+export interface RuleDescription {
+  /**
+   * Filter expression used to match messages. Supports 2 types:
+   * - `string`: SQL-like condition expression that is evaluated against the messages'
+   * user-defined properties and system properties. All system properties will be prefixed with
+   * `sys.` in the condition expression.
+   * - `CorrelationRuleFilter`: Properties of the filter will be used to match with the message properties.
+   */
+  filter?: string | CorrelationRuleFilter;
+  /**
+   * Action to perform if the message satisfies the filtering expression.
+   */
+  action?: string;
+  /**
+   * Represents the name of the rule.
+   */
+  name: string;
+}
+
+/**
+ * Represents the correlation filter expression.
+ * A CorrelationRuleFilter holds a set of conditions that are matched against user and system properties
+ * of incoming messages from a Subscription.
+ */
+export interface CorrelationRuleFilter {
+  /**
+   * Value to be matched with the `correlationId` property of the incoming message.
+   */
+  correlationId?: string;
+  /**
+   * Value to be matched with the `messageId` property of the incoming message.
+   */
+  messageId?: string;
+  /**
+   * Value to be matched with the `to` property of the incoming message.
+   */
+  to?: string;
+  /**
+   * Value to be matched with the `replyTo` property of the incoming message.
+   */
+  replyTo?: string;
+  /**
+   * Value to be matched with the `subject` property of the incoming message.
+   */
+  subject?: string;
+  /**
+   * Value to be matched with the `sessionId` property of the incoming message.
+   */
+  sessionId?: string;
+  /**
+   * Value to be matched with the `replyToSessionId` property of the incoming message.
+   */
+  replyToSessionId?: string;
+  /**
+   * Value to be matched with the `contentType` property of the incoming message.
+   */
+  contentType?: string;
+  /**
+   * Value to be matched with the user properties of the incoming message.
+   *
+   * Value can be of types string, number, boolean, Date or {val: string, kind: "duration"}
+   * Note: {val: string, kind: "duration"}
+   *       Meant for interoperability(to support the duration type in languages such as Java, .NET and Python)
+   *       `val` here is expected to be an ISO-8601 string representing the duration such as such as "PT1M" for 1 minute, "PT5S" for 5 seconds.
+   *            (More on ISO-8601 duration format: https://en.wikipedia.org/wiki/ISO_8601#Durations)
+   *       `kind` represents the "duration" type
```

Comment:
nit: the formatting of this comment looks off

Reply:
Fixed